bin/oak.rb

```diff
@@ -103,143 +103,141 @@
 
 =end
 
-if __FILE__ == $0
```

Comment:
The only change here was to get rid of the `if __FILE__ == $0` condition.

When `oak` has been installed as a gem, `bundler` creates a little forwarding script which gets put in the PATH:
```
hich oak
/Users/jhw/.rvm/gems/ruby-2.2.7/bin/oak
$ tail -3 `which oak`
gem "oak", version
load Gem.bin_path("oak", "oak", version)
end
```
When _this_ script is invoked from _that_ script, `$0` is something like `/Users/jhw/.rvm/gems/ruby-2.2.7/bin/oak` but `__FILE__` is something like `/Users/jhw/.rvm/gems/ruby-2.2.7/bin/oak.rb`.  They don't match, and so the script does nothing!

:(

Reply:
This file's change is easier to view in GitHub with `?w=1`.